src/app/views/app-sections/AppTitle.tsx

```diff
@@ -28,16 +30,27 @@ export function appTitleDisplayOnFullScreen(
     <div className={classes.graphExplorerLabelContainer} role={'heading'} aria-level={1}>
       {!minimised &&
         <>
-          {displayGraphLabel(classes)}
+          {displayGraphLabel(classes, permissionType)}
         </>}
     </div>
+    <div style={{ marginTop: 15 }}>
```

Comment:
at previous companies I worked at, there were style guidelines that restricted "magic number" css values (ie. all margin values need to be a multiple of 4px, for example). 
could you take a look at other divs/marginTops in the codebase and make sure that 15px is a value that's used often for margins?

Reply:
yup there are definitely magic values. i think the marginTop was added in for the toggle because the toggle takes up some space so ill remove all the newly added css floating around.